src/main/java/ca/corefacility/bioinformatics/irida/ria/web/cart/CartController.java

```diff
@@ -1,49 +1,32 @@
 package ca.corefacility.bioinformatics.irida.ria.web.cart;
 
-import ca.corefacility.bioinformatics.irida.model.project.Project;
-import ca.corefacility.bioinformatics.irida.model.sample.Sample;
-import ca.corefacility.bioinformatics.irida.ria.web.cart.components.Cart;
-import ca.corefacility.bioinformatics.irida.ria.web.cart.dto.*;
-import ca.corefacility.bioinformatics.irida.ria.web.oauth.GalaxyRedirectionEndpointController;
-import ca.corefacility.bioinformatics.irida.service.ProjectService;
-import ca.corefacility.bioinformatics.irida.service.sample.SampleService;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.beans.factory.annotation.Value;
-import org.springframework.context.annotation.Scope;
 import org.springframework.http.MediaType;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
-import org.springframework.web.bind.annotation.*;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
 
-import java.util.*;
+import ca.corefacility.bioinformatics.irida.ria.web.oauth.GalaxyRedirectionEndpointController;
 
 /**
  * Controller managing interactions with the selected sequences
  */
 @Controller
-@Scope("session")
 @RequestMapping("/cart")
 public class CartController {
-	private Cart cart;
-
-	private final SampleService sampleService;
-	private final ProjectService projectService;
-
 	@Value("${server.base.url}")
 	private String serverBaseUrl;
 	/*
 	 * Additional variables
 	 */
-	private String iridaPipelinePluginStyle;
+	private final String iridaPipelinePluginStyle;
```

Comment:
Remove the `final`

Reply:
Updated in 720b1f646043b2d45e37716fa49ddeb822744838